### ISSUE TITLE
[Frio] Change Nav Icon destinations

### ISSUE THREAD

**Issue by RiderExMachina** [friendica/friendica#12155]
### The What 
UX/UI related Feature change within the Frio theme. 

### They Why
In my opinion, the Nav bar does not fit with existing UX paradigms.

For example, the Home icon sends the user to their own profile. I found this confusing, as I would expect this to send me to my "home feed" of people I follow, rather than my own profile.

### The Change
My suggestion would be the following:
![Suggested Change](https://user-images.githubusercontent.com/8017578/201215260-4491814e-fbe7-41ca-a62d-417882b83451.png)

I would also recommend having Profile icon be the generic Person icon (see below) to fit with existing UX paradigms across the internet. ![Generic Person Icon](https://openclipart.org/image/800px/202776) to fit in.

I think these are all small things, and if given the go-ahead, I can start working on looking at the codebase to make these changes.

**Comment by MrPetovan** [comment]
Hi and thank you for your suggestion! Unfortunately this home icon has to stay as it is until we decide to remove the magic authentication feature. With this feature you are able to click on a Friendica profile link, and be authenticated on their own node where you can interact with posts there. So we need a button to return to your own node, and this is the purpose of the Home icon.

**Comment by RiderExMachina** [comment]
That sounds important. Is there any way we could keep that functionality and move it to a different icon that describes it better in visual language?

**Comment by MrPetovan** [comment]
We're slowly moving towards a model where users don't have to visit remote nodes so it's possible that this feature will be scrapped sooner than later. I'm wary of making any significant changes to the UI until then.

**Comment by RiderExMachina** [comment]
Understood, thank you for the reply.

Would it be helpful to keep this feature request open until then?

**Comment by MrPetovan** [comment]
Yes, but you can do even better, kick the hornet's nest by opening a new issue to remove/reduce the magic authentication feature, alongside a poll addressed to `!helpers@forum.friendi.ca` asking who's using the feature to interact with posts on remote nodes. 😈

But nobody must know it came from me! 🤫

**Comment by RiderExMachina** [comment]
I'll do what I can. Thank you for the information! :shushing_face:

**Comment by SpencerDub** [comment]
> With this feature you are able to click on a Friendica profile link, and be authenticated on their own node where you can interact with posts there.

I had _no_ idea this was a Friendica feature, and I've been using Friendica for years!

**Comment by MrPetovan** [comment]
Conceptually, it's pretty cool, but in practice the experience is inconsistent as you can end up on resource-starved nodes that are very slow to load, or be thrown into a completely different theme you are unfamiliar with. And it doesn't work with any other platform than Friendica, which breaks the behavior as a possible habit.

And of course it makes the code more complex, which for the little usage that it sees, isn't worthwhile.
